WinImageBuilder.psm1

```diff
@@ -1012,6 +1025,20 @@ function New-WindowsOnlineImage {
 }
 
 function New-WindowsCloudImage {
+    <#
+    .SYNOPSIS
+     This function creates a functional Windows Image, starting from an ISO file,
+     without the need of Hyper-V to be enabled.

```

Comment:
The image, to become ready for cloud usage, needs to be started on a hypervisor and it will automatically shut down when it finishes all the operations needed to become cloud ready: cloudbase-init installation, updates and sysprep.